src/index.html

```diff
@@ -15,8 +15,36 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/" class="logo__img"><img src="./images/logo.png" alt="Moyo"></a>
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="/apple" class="nav__link is-active">Apple</a>
+
+            <a href="/samsung" class="nav__link">Samsung</a>
+
+            <a href="/smartphones" class="nav__link">Smartphones</a>
+
+            <a href="/laptops_computers" class="nav__link"
```

Comment:
The text 'Laptops & Computers' in the anchor tag should be encoded as 'Laptops &amp; Computers' to ensure correct HTML entity encoding. The ampersand '&' character should be written as '&amp;' in HTML content.